webapp/src/js/services/db-sync.js

```diff
@@ -1,19 +1,12 @@
 var _ = require('underscore'),
-    READ_ONLY_TYPES = [ 'form', 'translations' ],
-    READ_ONLY_IDS = [ 'resources', 'appcache', 'zscore-charts', 'settings' ],
-    DDOC_PREFIX = [ '_design/' ],
-    META_SYNC_FREQUENCY = 30 * 60 * 1000; // 30 minutes
-
-angular.module('inboxServices').factory('DBSync',
-  function(
-    $interval,
-    $log,
-    $q,
-    Auth,
-    DB,
-    Session
-  ) {
+  READ_ONLY_TYPES = ['form', 'translations'],
+  READ_ONLY_IDS = ['resources', 'appcache', 'zscore-charts', 'settings'],
+  DDOC_PREFIX = ['_design/'],
+  META_SYNC_FREQUENCY = 30 * 60 * 1000; // 30 minutes
 
+angular
+  .module('inboxServices')
+  .factory('DBSync', function($interval, $log, $q, Auth, DB, Session) {
```

Comment:
Would we whitelist here?

Reply:
I don't think I'd whitelist this. I was never 100% happy with how it was. This change makes it a little hard to see which dependencies have been added or removed but only a little.

I think consistency wins here.